e2e-tests/portal/cypress/e2e/journeys/dashboard/dashboard-deals/filter-by/dashboard-deals-filter-by-notice-type.spec.js

```diff
@@ -77,12 +76,15 @@ context('Dashboard Deals filters - filter by submissionType/noticeType', () => {
     });
 
     it('renders only MIA deals', () => {
-      const ALL_MIA_DEALS = ALL_DEALS.filter(({ submissionType }) => submissionType === CONSTANTS.DEALS.SUBMISSION_TYPE.MIA);
-      dashboardDeals.rows().should('have.length', ALL_MIA_DEALS.length);
+      cy.login(BANK1_MAKER1);
```

Comment:
The test returns deals other than `MIA` when placed `beforeEach`

Reply:
This test only seems to test MIA so not sure why it can't be in a beforeEach? Something is wrong with the filtering then if it returning other deals which are not MIA.  Otherwise this could be a shared cypress command